pkg/pillar/cmd/domainmgr/domainmgr.go

```diff
@@ -462,6 +463,9 @@ func gcObjects(ctx *domainContext, dirName string) {
 		}
 		unpublishImageStatus(ctx, &status)
 	}
+
+	// Run rkt garbage collect
+	rktGc(ctx.RktGCGracePeriod)
```

Comment:
You don't check for error return; should decide whether function returns error and caller logs, or function doesn't return an error and logs itself.

Reply:
Good Catch.. No point returning.. Changed prototype to not return..